shared/base/src/main/java/com/ivy/base/time/impl/StandardTimeConvert.kt

```diff
@@ -0,0 +1,28 @@
+package com.ivy.base.time.impl
+
+import com.ivy.base.time.TimeConverter
+import com.ivy.base.time.TimeProvider
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import javax.inject.Inject
+
+class StandardTimeConvert @Inject constructor(
```

Comment:
We should gracefully handle the MIN and MAX cases of time. Under them the app will currently crash which isn't desired. The MIN/MAX is likely to happen if we're importing data from 3rd party apps.

Reply:
Will be fixed in a follow-up PR